src/app.module.ts

```diff
@@ -45,6 +47,10 @@ import { RedisModule } from './shared/auth/miscs/redis.module';
         path: 'company',
         module: CompanyModule,
       },
+      {
+        path: 'resume',
+        module: ResumesModule,
+      }
```

Comment:
nope, nope 

we use the router module to make the professionals and company isolated,

resume should live under the professionals